src/typings.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module 'styled-components' {
```

Comment:
why is this in a seperate file?

Reply:
I tried with @types/styled-components which does not work for me. So I created it by myself.

Reply:
It has to be in a .d.ts file, as far as know

Reply:
I don't understand what is going on here. You are declaring the types for the styled-components package? What happens without this? Does typescript complain?

Reply:
yes, typescript complains.
So I started with installing @types/styled-components which does not work with rtlPluginSc from 'stylis-plugin-rtl-sc'. 
As rtlPluginSc only takes `(context: number, content: string)`.
So I solved it by declaring the types for the styled-components package by myself.